sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryAsyncClient.java

```diff
@@ -302,13 +302,13 @@ public DirectoryAsyncClient getSubDirectoryClient(String subDirectoryName) {
      * @param recursive Specifies operation should apply to the directory specified in the URI, its files, its subdirectories and their files.
      * @return {@link HandleItem handles} in the directory that satisfy the requirements
      */
-    public Flux<HandleItem> getHandles(Integer maxResult, boolean recursive) {
+    public Flux<HandleItem> listHandles(Integer maxResult, boolean recursive) {
         return azureFileStorageClient.directorys().listHandlesWithRestResponseAsync(shareName, directoryPath, null, maxResult, null, snapshot, recursive, Context.NONE)
                    .flatMapMany(response -> nextPageForHandles(response, maxResult, recursive));
     }
 
     /**
-     * Closes a handle or handles opened on a directory or a file at the service. It is intended to be used alongside {@link DirectoryAsyncClient#getHandles(Integer, boolean)} .
+     * Closes a handle or handles opened on a directory or a file at the service. It is intended to be used alongside {@link DirectoryAsyncClient#listHandles(Integer, boolean)} .
```

Comment:
Avoid having TODOs in public API javadocs. It will show up on our apidocs.

https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/com/azure/storage/file/DirectoryClient.html#forceCloseHandles-java.lang.String-boolean-

Reply:
Jonathan suggested us to leave issue here in TODO for open comments. Will do a clean up before GA or before release.

Reply:
Maybe move the TODO comments inside the method and put it in a block comment? I am not sure if @JonathanGiles intended the TODO to be in the Javadoc visible to public.

Reply:
I can move todo into method.